### PR TITLE
Output timestamps

### PR DESCRIPTION
**Pull request by missytake**
For a more verbose log output. Timestamps are useful at debugging failed login attempts.

### CODE REVIEW

**Comment by missytake** [comment]
I deployed this branch to login.testrun.org, it seems to work :)

**Review by pabzm — CHANGES_REQUESTED** [review]
Looks good, thanks!

But 53d174b2be2ff19bb9380f6ee0968c7f74192ab4 ("update core") is not related to this change, please remove that commit.

**Comment by missytake** [comment]
Isn't the state of package-lock.json with which the changes worked important? I don't know much about node.js, but at desktop I was once asked to build a GitHub action to specifically remind developers that they should push the results of `npm i` as well... just interested :)

**Comment by pabzm** [comment]
Yes, please include `package-lock.json`. But you changed the versions of DC core, too. And that change does not belong into one PR with introducing timestamps —  unless there is a technical dependency, which I'm unaware of.

**Comment by missytake** [comment]
okay. do we still want to upgrade core afterwards though?

**Comment by missytake** [comment]
I think we need to do it for #5 anyway

**Comment by pabzm** [comment]
> okay. do we still want to upgrade core afterwards though?

I guess so, yes. But I'm trying to push for clean, small PRs, that cover one topic only.

File: package.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
   "dependencies": {
     "body-parser": "^1.19.0",
     "cookie-parser": "^1.4.4",
-    "deltachat-node": "^1.0.0-beta.15",
+    "deltachat-node": "1.0.0-alpha.11",
```
Comment (pabzm) [inline package.json:15]: The master-branch has `^1.0.0-beta.15`, so that shouldn't be changed in this PR.

File: package-lock.json [hunk 2]
```diff
@@ -308,7 +308,7 @@
     "deltachat-node": {
       "version": "1.0.0-alpha.11",
       "resolved": "https://registry.npmjs.org/deltachat-node/-/deltachat-node-1.0.0-alpha.11.tgz",
-      "integrity": "sha1-Y8J2krXuyrOBxUFZxG6HGWa5FDY=",
+      "integrity": "sha512-L9yisI+xd/mLEQEsg1Or+CwdXFkBFr60zBC22sfPef0gedTmtQc0yRUFTRVGwbTTHvDenEUwx9P7fkwSh2eWKA==",
```
Comment (pabzm) [inline package-lock.json:311]: This shouldn't be changed in this PR.

**Review by pabzm — CHANGES_REQUESTED** [review]
After fixing the core-version strings, could you clean the commit history so that only the necessary commits stay in the history? E.g. drop both the commits that change and revert a thing.

**Comment by Simon-Laux** [comment]
Tip: you can also replace the log function to sth. like:

```js
function log (...args) {
console.log(`[${(new Date()).toLocaleString("en-US")}]`,...args);
}
```

**Comment by missytake** [comment]
closing this in favor of #10

**Pull request closed without merging**